### DISCUSSION TITLE
Vehicle communication protocol

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#127] [General]
Does the car communication protocol Modbus be supported？

**Comment by swltr** [comment]
It is unclear what you mean here. In case you mean to ask whether Modbus can be used as a protocol to communicate between vehicle driver and vehicle, then the answer is that _any_ communication protocol can be used as long as you implement the respective driver.

**Comment by HuJunnj** [comment]
okay l see, thanks

best regards
Jun



---- Replied Message ----
| From | Stefan ***@***.***> |
| Date | 02/06/2024 15:50 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [openTCS/opentcs] Vehicle communication protocol (Discussion #127) |

It is unclear what you mean here. In case you mean to ask whether Modbus can be used as a protocol to communicate between vehicle driver and vehicle, then the answer is that any communication protocol can be used as long as you implement the respective driver.
